lua/pixelui/elements/cl_sidebar.lua

```diff
@@ -36,22 +36,28 @@ end
 function PANEL:Paint(w, h)
     local textCol = PIXEL.Colors.SecondaryText
     local backgroundCol = PIXEL.Colors.Transparent
+    local hoverStyleCol = PIXEL.Colors.Transparent
```

Comment:
I don't understand your confusion with naming this variable. It's the colour of a line, `lineCol`?

Reply:
omfg